dev-proxy/ProxyHost.cs

```diff
@@ -416,7 +417,26 @@ public RootCommand GetRootCommand(ILogger logger)
         jwtValidForOption.AddAlias("-v");
         jwtCreateCommand.AddOption(jwtValidForOption);
 
-        jwtCreateCommand.SetHandler( 
+        var jwtSigningKeyOption = new Option<string>("--signingkey", "The signing key to sign the token. Minimum length is 32 characters.");
```

Comment:
Let's follow our naming convention of separating words with `-`

```suggestion
        var jwtSigningKeyOption = new Option<string>("--signing-key", "The signing key to sign the token. Minimum length is 32 characters.");
```